apps/meteor/server/models/raw/LivechatRooms.ts

```diff
@@ -1130,8 +1130,8 @@ export class LivechatRoomsRaw extends BaseRaw<IOmnichannelRoom> implements ILive
 		const match: Document = {
 			$match: {
 				'v._id': visitorId,
-				...(open !== undefined && { open: { $exists: open } }),
-				...(served !== undefined && { servedBy: { $exists: served } }),
+				...(open !== undefined && !open && { closedAt: { $exists: true } }),
```

Comment:
What if I want to select the open chats only? Should they need to have `open: true` explicitly? Is there any possibility of having a room that was not closed (`closedAt`) but also not `open`? 🤔